r/R/K_functions.R

```diff
@@ -2,195 +2,206 @@
 #'
 #' @param sal Salinity
 #' @return Ionic strength
-fn_Istr <- function(sal) {
+calc_ionic_strength <- function(sal) {
   Istr <- 19.924 * sal / (1000 - 1.005 * sal)
 
   return(Istr)
 }
 
 #' Calculate K1
 #'
-#' @param p Parameters for K calculation
-#' @param temp_k Temperature (Kelvin)
+#' @param coefficients Coefficients for K calculation
+#' @param temp_c Temperature (Celcius)
 #' @param sal Salinity
 #' @return K1
-fn_K1 <- function(p, temp_k, sal) {
-  K1 <- 10^(p[1] +
-    p[2] / temp_k +
-    p[3] * log(temp_k) +
-    p[4] * sal +
-    p[5] * sal^2)
+calc_K1 <- function(coefficients, temp_c, sal) {
+  temp_k <- temp_c+273.15
```

Comment:
@rossidae We need to do some styling. `usethis::use_tidystyle()`

Reply:
Ahhh gotcha - just worked out how to enable this in VSCode.
Newest commit should solve most of these. I will break up the remaining overlong lines as well.